Cargo.toml

```
@@ -66,19 +68,25 @@ tokio = { workspace = true, features = ["test-util"] }
 
 [features]
 systemd = ["dep:systemd"]
+forwarder = ["dep:edgehog-forwarder", "dep:edgehog-device-forwarder-proto"]
 e2e_test = []
 
 [workspace.dependencies]
 astarte-device-sdk = "0.5.1"
 astarte-message-hub = "0.5.2"
 async-trait = "0.1.72"
+backoff = "0.4.0"
 base64 = "0.13.1"
 bollard = "0.15.0"
 bytes = "1.5.0"
 clap = "4.3.19"
 displaydoc = "0.2.4"
+edgehog-device-forwarder-proto = { git = "https://github.com/edgehog-device-manager/edgehog-device-forwarder-proto.git" }
```

Change to `alpha` release published on `crates.io`.
```suggestion
edgehog-device-forwarder-proto = "0.1.0-alpha.0"
```

---

Thanks for the reminder. I already changed it but it seems i undid the changes somehow.

---

Done